docs/machine-learning/tutorials/object-detection-onnx.md

```diff
@@ -0,0 +1,1124 @@
+
+![](./media/object-detection-onnx/img-classification-obj-detection.PNG)
+
+Some use cases for object detection include:
+
+- Self-Driving Cars
+- Robotics
+- Face Detection
+- Workplace Safety
+- Object Counting
+- Activity Recognition
+
+## Select the appropriate machine learning task 
+
+Deep learning is a subset of machine learning. To train deep learning models, large quantities of data are required. Patterns in the data are represented by a series of layers. The relationships in the data are encoded as connections between the layers contanining weights. The higher the weight, the stronger the relationship. Collectively, this series of layers and connections are known as artificial neural networks. The more layers in a network, the "deeper" it is, making it a deep neural network. 
+
+There are different types of neural networks, each more suitable than the other for certain tasks. The most common types of neural networks are Multi-Layered Perceptron (MLP), Convolutional Neural Network (CNN) and Recurrent Neural Network (RNN). The most basic is the MLP which maps a set of inputs to a set of outputs. This neural network is good when the data does not have a spatial or time component. The CNN as implied by the name makes use of convolutional layers which take into account spatial information contained in the data. A good use case for CNNs is image processing when trying to detect the presence of a particular feature in a region of an image (i.e. is there a nose in the center of an image?). Finally, RNNs allow for the persistence of state or memory to be used as input therefore making it a good candidates for time-series analysis where the sequential ordering and context of events is important. 
+
+Object detection is an image processing task. Therefore, most deep learning models trained to solve this problem are CNNs. The model used in this tutorial is the Tiny YOLOv2 model, a more compact version of the YOLOv2 model described in the paper: ["YOLO9000: Better, Faster, Stronger" by Redmon and Fadhari](https://arxiv.org/pdf/1612.08242.pdf). Tiny YOLOv2 is trained on the Pascal VOC dataset and is made up of 15 layers that can predict 20 different classes of objects. Because Tiny YOLOv2 is a condensed version of the original YOLOv2 model, a tradeoff is made between speed and accuracy. 
+
+The model is stored in ONNX format. The Open Neural Network Exchange (ONNX) is an open source format for AI models. ONNX supports interoperability between frameworks. This means you can train a model in one of the many popular machine learning frameworks like PyTorch, convert it into ONNX format and consume the ONNX model in a different framework like ML.NET. To learn more, visit the [ONNX website](https://onnx.ai/). 
+
+![](./media/object-detection-onnx/onnx-frameworks.png)
```

Comment:
A diagram with the network inputs and outputs would be very useful here. If you draft it, we can ask Deborah in art to make it professional